contracts/access/IAccessControl.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+// From OpenZeppelin Contracts v4.4.1 (access/IAccessControl.sol)
+
+pragma solidity ^0.8.0;
+
+import { IAccessControlInternal } from './IAccessControlInternal.sol';
+
+/**
+ * @dev External interface of AccessControl declared to support ERC165 detection.
+ */
+interface IAccessControl is IAccessControlInternal {
+    /**
```

Comment:
Follow natspec best practices for the entire file.

Reply:
this is a copy of OpenZeppelin, which means OpenZeppelin doesn't follow natspec best practices??

Reply:
@NouDaimon SolidState hasn't settled on best practices for contract- and interface-level comments.  We'll enforce this at a later date.

@iam-dev No action needed here.

Reply:
>> OpenZeppelin doesn't follow natspec best practices??

What @NouDaimon is referring to is the "best practice" of making full use of all available tags.  OZ doesn't really do this - for example, in addition to `dev`, here they could use `title` and `author`, and maybe more.

Reply:
@iam-dev 'best-practices' are kind of vague, indeed; as far as I know, it is best to be as clear as possible. Therefore, as many tags as apply is the rule of thumb I go by - it is also what is done in SS :) Look at:
https://github.com/solidstate-network/solidstate-solidity/blob/master/contracts/token/ERC20/IERC20.sol

Reply:
Generally I use the `title` tag to briefly describe the contract, plus a `dev` tag is there's additional miscellaneous information.  But it's only about 80% consistent across the library.  A future PR will fix, and from that point on we will enforce consistency.